### PR TITLE
Add React Router and the basic Navigation

### PR DESCRIPTION
**Pull request by gissellediaz**
# Description

Add React Router and the basic Navigation 

Fixes # https://trello.com/c/tKHXSNHG/9-add-react-router-and-the-basic-navigation)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/pages/home/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Home from './home';
+
+export default Home;
```
Comment (davidfigueroar9) [inline src/pages/home/index.js]: we need a new line at the end of file.

File: src/pages/home/components/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Header from './header';
+
+export {
+  Header,
+};
+
+export default {
+  Header,
+};
```
Comment (davidfigueroar9) [inline src/pages/home/components/index.js]: we need a new line at the end of file.

File: src/pages/explore/explore.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Explore = () => (
+  <h2>Explore</h2>
+);
+
+export default Explore;
```
Comment (davidfigueroar9) [inline src/pages/explore/explore.jsx]: we need a new line at the end of file.

File: src/components/iconLink/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import IconLink from './iconLink';
+
+export default IconLink;
```
Comment (davidfigueroar9) [inline src/components/iconLink/index.js]: we need a new line at the end of file.

File: src/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  Likes,
+  Home,
+};
+
+export default {
+  Explore,
+  Profile,
+  Likes,
+  Home,
+};
```
Comment (davidfigueroar9) [inline src/pages/index.js]: we need a new line at the end of file.

File: src/pages/likes/index.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import Likes from './likes';
+
+export default Likes;
```
Comment (davidfigueroar9) [inline src/pages/likes/index.js]: we need a new line at the end of file.

File: src/pages/likes/likes.jsx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Likes = () => (
+  <h2>Likes</h2>
+);
+
+export default Likes;
```
Comment (davidfigueroar9) [inline src/pages/likes/likes.jsx]: we need a new line at the end of file.

File: src/pages/profile/index.js [hunk 8]
```diff
@@ -0,0 +1,3 @@
+import Profile from './profile';
+
+export default Profile;
```
Comment (davidfigueroar9) [inline src/pages/profile/index.js]: we need a new line at the end of file.

File: src/pages/profile/profile.jsx [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Profile = () => (
+  <h2>Profile</h2>
+); 
+
+export default Profile;
```
Comment (davidfigueroar9) [inline src/pages/profile/profile.jsx]: we need a new line at the end of file.

File: src/pages/home/index.js (hunk 1 above)
Comment (gissellediaz) [reply src/pages/home/index.js]: I just fixed it!

File: src/pages/home/components/index.js (hunk 2 above)
Comment (gissellediaz) [reply src/pages/home/components/index.js]: done!

File: src/pages/explore/explore.jsx (hunk 3 above)
Comment (gissellediaz) [reply src/pages/explore/explore.jsx]: fixed.

File: src/components/iconLink/index.js (hunk 4 above)
Comment (gissellediaz) [reply src/components/iconLink/index.js]: Done.

File: src/pages/index.js (hunk 5 above)
Comment (gissellediaz) [reply src/pages/index.js]: done!

**Pull request merged**